pyproject.toml

```diff
@@ -283,6 +283,7 @@ max-returns = 10
     "PT004",    # fixture ignore type returning.
     "E501",     # Test skips lines that are too long.
     "ANN001",   # Skip annotation type hint in test codes
+    "ANN201",   # Skip return type hint in test codes
```

Comment:
This might need the concensus since we've been implementing the test code with the type hint. @harimkang, @sovrasov  what do you think ?

Reply:
@vinnamkim , @samet-akcay

Reply:
It would be nice to have a consensus for the test code as well, but I don't know if it's necessary for the test code. Also, the return type is mostly None in test code, so I think it's okay to ignore it.

Reply:
yes @harimkang is right. Number of HPO unit test function is hundreds and honestly, adding `-> None` is meaningful.

Reply:
For the functions, I agree. However, I think for the variable or argument, type-hint is helpful to understand what variable means for me.

Reply:
yes but what I added is about return type. if we remove `ANN001`, we may need to change overall test code. I think it's better to discuss about it later.

Reply:
Ah, sorry, I just missed the information. it just represents the return_type.